integration-tests/src/test/java/org/zowe/apiml/util/config/EnvironmentConfiguration.java

```diff
@@ -27,7 +29,8 @@ public class EnvironmentConfiguration {
     private ApiCatalogServiceConfiguration apiCatalogServiceConfiguration;
     private ApiCatalogServiceConfiguration apiCatalogStandaloneConfiguration;
     private CachingServiceConfiguration cachingServiceConfiguration;
-    private CloudGatewayConfiguration cloudGatewayConfiguration;
+    // Cloud gateway tests are excluded from z/OS tests, leading to config load failure when the props are undefined
+    private CloudGatewayConfiguration cloudGatewayConfiguration = new CloudGatewayConfiguration();
```

Comment:
won't this prevent the github action from running? is the ignore unknown not enough?